wegodang/users/validator.py

```diff
@@ -0,0 +1,8 @@
+import re
+
+from django.core.exceptions import ValidationError
+
+def validate_email(email):
+    EMAIL_REGEX = re.match('^[a-zA-z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9.]+$', email)
+    if not EMAIL_REGEX:
```

Comment:
사소 할 수 있지만, 구문 사이에 공백 추가해 주세요.
```suggestion
    EMAIL_REGEX = re.match('^[a-zA-z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9.]+$', email)
    
    
    if not EMAIL_REGEX:
```
```suggestion
    EMAIL_REGEX = re.match('^[a-zA-z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9.]+$', email)
    if not EMAIL_REGEX:
```

Reply:
수정하였습니다 !